lib/mixins/helpers.js

```diff
@@ -119,31 +119,18 @@ var helpers = {
   },
   getSlideClasses: function (index) {
     var slickActive, slickCenter, slickCloned;
-    var centerOffset, indexOffset;
-    var allSlides;
-    var centerIndex;
-    var realRange = false;
-
+    var currentSlide = this.state.currentSlide;
+    var centerOffset;
 
     slickCloned = (index < 0) || (index >= this.state.slideCount);
     if (this.props.centerMode) {
-      if (this.refs.track) {
```

Comment:
@akiran I simplified the implementation here and removed all the unnecesary variables. Have a look at it.